### DISCUSSION TITLE
Traducción de enlace (link) externo de vídeo que no está en español

### DISCUSSION THREAD

**Discussion by AFBGdev** [reactjs/es.reactjs.org#695] [Q&A] [answered]
Estoy traduciendo la página [/community/videos](https://es.react.dev/community/videos) y está esta sección (Imagen adjunta) donde hay enlaces a videos de YouTube que no estan en español.

![image](https://github.com/reactjs/es.react.dev/assets/22183829/d1957ccb-a15f-479b-b707-1988e1a674e0)

Mi pregunta es:

¿Debería traducir los títulos de los videos y dejar los enlaces en ingles o debería dejar tanto el título como el enlace en ingles?

Sé que hay una [guía de estilo para enlaces externos](https://github.com/reactjs/es.react.dev/wiki/Gu%C3%ADa-de-estilo#enlaces-externos), pero; no hay un ejemplo claro con éste caso.

De antemano, muchas gracias!.

**Comment by carburo** [comment, marked as answer]
Puede tener valor traducir el título, sobre todo porque veo que algunos videos tienen subtítulos en español (no autogenerados). Creo que también sería conveniente especificar esta información junto al título. 
Por ejemplo: Aprender en el navegador (en inglés, subtítulos en español).
Es solo mi opinión, son bienvenidos otros puntos de vista.

**Reply by Byron2016** [reply]
Propongo que en la [Guia de estilo ](https://github.com/reactjs/es.react.dev/wiki/Gu%C3%ADa-de-estilo) se lo agregue a fin que los títulos sean traducidos y poner agregar entre paréntesis según lo sugerido y si el link está en inglés, colocar (inglés) entre paréntesis.
